src/index.js

```diff
@@ -0,0 +1,42 @@
+import './index.scss';
+
+const listContainer = document.querySelector('.todo__main-taskContainer');
+const refreshButton = document.querySelector('.todo__header-refreshButton');
+
+const todosTaskList = [
+  {
+    description: 'Morning walk at 5AM with brother',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Go to GYM for weight loss',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Buy some groceries from city',
+    completed: false,
+    index: 3,
+  },
+];
+
+todosTaskList.forEach((each) => {
+  const htmlTemplateLiteral = `
+    <li class="space todo__list" id="list-${each.index}" draggable="true" onclick="todoListHandler(event, this)">
+      <input class="todo__list-status" type="checkbox" name="todo__task_checkbox" ${each.completed ? 'checked' : ''} id="todo__task_checkbox-${each.index}" aria-label="mark the task as completed or uncompleted" >
+      <label for="task-${each.index}" class="todo__task-placeholder" onclick="todoEditHandler(this)">
+        <span class="strike">${each.description}</span>
+        <input class="update-todo__input" type="text" name="task-${each.index}" id="task-${each.index}" >
+      </label>
+      <div class="todo__mode">
+        <i class="fontawesome-icon fa-solid fa-ellipsis-vertical ellipse" id="todo__mode-${each.index}"></i>
+      </div>
+    </li>
+  `;
+  listContainer.insertAdjacentHTML('beforeend', htmlTemplateLiteral);
+});
+
+refreshButton.addEventListener('click', () => {
+  window.location.reload();
+});
```

Comment:
- [ ] Good job so far your project is well implemented :+1:  But from the project requirement, the list should appear in order of the index values for each task but it is not the case in your project when I change the index value  :smile: Please fix it . :+1: 
![Screenshot from 2022-08-10 20-17-35](https://user-images.githubusercontent.com/6615394/184012076-15c6e7ef-e1e1-4561-9818-2e1dd7ddef49.png)

Reply:
@codecaiine Thank you so much. I will update it ASAP.